src/builder/components/question/card/survey-question-card.component.tsx

```diff
@@ -0,0 +1,99 @@
+import { message } from 'antd';
+import React, { Component } from 'react';
+import { SurveyItem } from '../../../../common/types/survey-item.type';
+import { EditableTitle } from '../editable-title/survey-question-editable-title.component';
+import { SurveyQuestionHighlight } from '../highlight/survey-question-highlight.component';
+import { SurveyQuestion } from '../survey-questions/survey-question.component';
+import { SurveyQuestionToolbox, ToolboxAction } from '../toolbox/survey-question-toolbox.component';
+import './survey-question-card.component.less';
+
+export interface SurveyQuestionCardProps {
+    question: SurveyItem;
+    onDelete: (item: SurveyItem) => void;
+    onUpdate: (item: SurveyItem) => void;
+}
+
+export interface SurveyQuestionCardState {
+    toolboxVisible?: boolean;
+    nameEditing: boolean;
+}
+
+export class SurveyQuestionCard extends Component<SurveyQuestionCardProps, SurveyQuestionCardState> {
+    state = {
+        toolboxVisible: false,
+        nameEditing: false,
```

Comment:
What's this field for?

Reply:
An unfortunate artifact of refactoring. Removed it.